1-exercises/D-for-in-loops/exercise-1.js

```diff
@@ -13,6 +13,15 @@ let salaries = {
 };
 
 // WRITE BELOW THIS LINE
+for (let property in salaries){
+if (salaries[property] > 50000) {
+  //console.log(property);  
+  console.log(property + ": $" + salaries[property]);
+}
```

Comment:
Nit: this should be indented. Also try to remember to remove commented code before you submit 👌